web/client/plugins/FeatureEditor.jsx

```diff
@@ -269,6 +269,16 @@ const EditorPlugin = compose(
         componentDidMount() {
             // only the passed properties will be picked
             this.props.onMount(pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']));
+        },
+        // TODO: fix this in contexts
+        // due to multiple renders of plugins in contexts (one with default props, then with context props)
+        // the options have to be updated when change.
+        componentDidUpdate(oldProps) {
+            const newOptions = pick(this.props, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
+            const oldOptions = pick(oldProps, ['showFilteredObject', 'showTimeSync', 'timeSync', 'customEditorsOptions']);
+            if (JSON.stringify(newOptions) !== JSON.stringify(oldOptions) ) {
```

Comment:
why stringify and not !isEqual or deepEqual from lodash?

Reply:
because it's faster comparison . https://www.mattzeunert.com/2016/01/28/javascript-deep-equal.html#:~:text=Another%20difference%20between%20the%20two,stringify%20does%20not%20serialize%20functions.&text=deepEqual%20will%20instead%20check%20for,counterpart%20in%20the%20other%20object.

Reply:
ok :)